README.md

```diff
@@ -2,7 +2,149 @@
+
+Jitsi will become available at the designated hostname after a few minutes.
+
+## Meet
+
+Connect to the host via `https://<hostname>` and start a meeting.
+
+By default, authentication is enabled - though unauthenticated guests can enter after
+an authenticated host has started a meeting.
+The default authenticated user name is `flatcar`, and the password is auto-generated on
+the node during provisioning.
+Initial user and password are appended to `/opt/jitsi/.env` on the server.
+See Customisation below on how to pass a custom user/password into the deployment.
+
+## Customisations
+
+- Installer script settings can be overridden in [jitsi-install.env](jitsi-install.env):
+  - `MODERATOR_USER` Custom Jitsi meeting moderator user name
+  - `MODERATOR_PASS` Password for `MODERATOR_USER`.
+     Will be auto-generated at provisioning time and appended to `/opt/jitsi/.env` if not set.
+  - `DEPLOY_SET_PUBLIC_IP` When `true`, before installing Jitsi, determine the server's public IP
+     address and explicitly configure Jitsi for that IP address (via `JVB_ADVERTISE_IPS`).
+     This is handy for running in a NATed / load-balanced cloud environment where the 
+     instance doesn't know its public IP address (e.g. in protected Azure environments).
+     Alternatively, this can be set to a public IP address to use _that_ address for the server.
+     _Set to empty or to `false` to disable._
+  - `DEPLOY_WAIT_FOR_HOSTNAME_DNS` wait for the hostname `JITSI_SERVER_FQDN` to resolve to
+    the host's public IP before installing Jitsi. Needs `DEPLOY_SET_PUBLIC_IP`.
+    In order to ensure we're not flooding LetsEncrypt with certificate queries (and get blocked as a result), the
+    installer will check whether the designated hostname matches its public IP address before
+    continuing the installation.
+- Jitsi settings such as LetsEncrypt usage, authentication, live streaming and recording can be set in
+  `jitsi-config.env`. All env variables of jitsi-docker's
+  [`docker-compose.yml`](https://github.com/jitsi/docker-jitsi-meet/blob/master/docker-compose.yml)
+  and [`jibri.yaml`](https://github.com/jitsi/docker-jitsi-meet/blob/master/jibri.yml)
+  can be overridden there.
+
+### ARM64
+
+If you want to deploy to an ARM64 server you'll need to change [`config.yaml`](config.yaml) to use
+ARM64 sysexts for docker and docker compose. See the sysexts' `target:` lines in the `links:` section
+and the `- path:` and `source:` lines in the `files:` section respectively.
+
+## Test the set-up locally
+
+You can test locally in a qemu VM.
+Uncomment the local test settings at the bottom of [`jitsi-config.env`](jitsi-config.env),
+disable `DEPLOY_SET_PUBLIC_IP` in [jitsi-install.env](jitsi-install.env),
+and re-generate the configuration.
+
+Fetch the latest `flatcar_production_qemu_uefi_efi_code.fd`,
+ `flatcar_production_qemu_uefi_efi_vars.fd`,
+ `flatcar_production_qemu_uefi_image.img`,
+ `flatcar_production_qemu_uefi_image.img.orig`, and
```

Comment:
Lol, that's a copy-paste error from `ls -1` in my dev env. No need to make a copy since we're using `-snapshot`. Good catch! Removed the spurious file.